src/input/hotkey.c

```diff
@@ -160,10 +166,58 @@ static void input_number(int number)
     }
 }
 
+static void new_map(void)
+{
+    if (editor_is_active() && window_is(WINDOW_EDITOR_MAP)) {
+        // TODO: here we need the size selection for triggering window_select_list_show?
+    } else if (window_is(WINDOW_CITY)) {
+        building_construction_clear_type();
+        game_undo_disable();
+        game_state_reset_overlay();
+        window_main_menu_show(1);
+    }
+}
+
+static void load_file(void)
+{
+    if (editor_is_active() && window_is(WINDOW_EDITOR_MAP)) {
+        window_editor_map_show();
```

Comment:
Since you're already on the editor map, there's no need to show it again, you can remove this line.